promexporter/exporter.go

```diff
@@ -16,15 +16,19 @@ var (
 	promPort          string
 	promEndpoint      string
 	promTLSConfigPath string
+	hideIP            bool
```

Comment:
It makes sense to name variables like this: `metricHideIP`, `metricHideUser`. See next comment.

Reply:
I renamed them.